docs(sources): add integrations directory with 31 source pages (ETL-980) (#739)
glassflow/clickhouse-etl: docs/app/sources/fluent-bit/page.mdx

---
title: 'Fluent Bit'
description: 'Forward logs and metrics from Fluent Bit into ClickHouse with GlassFlow'
---
import { Callout } from 'nextra/components'

# Fluent Bit

Fluent Bit is a lightweight, high-performance log and metrics collector designed for cloud-native environments. It collects, processes, and forwards telemetry data from sources like containers and systems to destinations such as Kafka or ClickHouse.

## Using Fluent Bit with GlassFlow

GlassFlow can ingest telemetry forwarded by Fluent Bit into ClickHouse with the same pipeline capabilities available to all GlassFlow sources — deduplication, filtering, and stateless transformations.

<Callout type="info">

> **Available on request.** Fluent Bit connectors are supported natively via the GlassFlow Enterprise Edition. To get started, [join our GlassFlow Slack community](https://glassflowhub.slack.com/join/shared_invite/zt-349m7lenp-IFeKSGfQwpJfIiQ7oyFFKg) to chat directly with our team, or [reach out](https://www.glassflow.dev/integrations#contact) to us with your use case, expected volumes, and timeline.

Reviewer: ```suggestion
**Available on request.** Fluent Bit connectors are natively supported in GlassFlow Enterprise Edition. To get started, [join our GlassFlow Slack community](https://glassflowhub.slack.com/join/shared_invite/zt-349m7lenp-IFeKSGfQwpJfIiQ7oyFFKg) to chat directly with our team, or [reach out](https://www.glassflow.dev/integrations#contact) to us with your use case, expected volumes, and timeline.
```
